lib/FindTests.js

```diff
@@ -0,0 +1,263 @@
+// @flow
```

Comment:
Are there any changes to these functions, or have they just been moved?

Reply:
A lot of the diff is actually just moved stuff! I’ve added comments to show what is just moved and what has changed.